roles/install-k8s/tasks/main.yml

```diff
@@ -67,7 +67,7 @@
   when: node_type == "master"
 
 - name: remove taint
-  command: "kubectl taint nodes --all node-role.kubernetes.io/master-"
+  command: "kubectl taint nodes --all node-role.kubernetes.io/control-plane-"
```

Comment:
I see below taints in latest k8s cluster:
```
Taints:             node-role.kubernetes.io/control-plane:NoSchedule
                    node-role.kubernetes.io/master:NoSchedule
```
We may have to remove both the taints to make mater node schedulable.

Reply:
validated and fixed.

Reply:
@Rajalakshmi-Girish will this `kubectl taint nodes --all node-role.kubernetes.io/master-` work? because

```
fatal: [158.175.162.83]: FAILED! => {"changed": true, "cmd": "kubectl get node --selector='node-role.kubernetes.io/master' --no-headers | awk '{print $1}' | xargs kubectl cordon", "delta": "0:00:00.103623", "end": "2022-03-07 09:20:09.189542", "msg": "non-zero return code", "rc": 123, "start": "2022-03-07 09:20:09.085919", "stderr": "No resources found\nerror: USAGE: cordon NODE [flags]\nSee 'kubectl cordon -h' for help and examples", "stderr_lines": ["No resources found", "error: USAGE: cordon NODE [flags]", "See 'kubectl cordon -h' for help and examples"], "stdout": "", "stdout_lines": []}
[791](https://prow.ppc64le-cloud.org/view/gs/ppc64le-kubernetes/logs/periodic-kubernetes-storage-perf-test-ppc64le/1500755896580444160#1:build-log.txt%3A791)
```

Reply:
@piyushgupta1551 ^^

Reply:
@mkumatag , The update includes the selector label change as well and it works fine. See below - 
```
[root@kubetest2-cluster-master ~]# kubectl get node --selector='node-role.kubernetes.io/control-plane' --no-headers | awk '{print $1}' | xargs kubectl cordon
node/kubetest2-cluster-master cordoned
```

Reply:
@mkumatag The latest clusters don't have the label `node-role.kubernetes.io/master` but the taints section shows a taint `node-role.kubernetes.io/master:NoSchedule`
@piyushgupta1551 validated these commands on a cluster that was recently created.

Reply:
Let's merge this then..